code-challenges.js

```diff
@@ -9,67 +9,156 @@
 
-var myMapNumbers1 = [87, 6, 28, 16, 2]
-var myMapNumbers2 = [8, -7, 0, 6, 12]
+// write the variables
+// var myMapNumbers1 = [87, 6, 28, 16, 2]
+// var myMapNumbers2 = [8, -7, 0, 6, 12]
+// // write the function that will pull each number and multiply it by 5 using map
+// // const mappedNums = myMapNumbers1.map(numbers =>  {
+// //       return numbers * 5
+// //         // get the return and multiply it by 5
+// // })
 
+// // console.log(mappedNums)
+// // I DID IT WITH ONE ARRAY BUT CANT FIGURE OUT HOW TO GET BOTH, LET ME TRY AGAIN
 
+// const  mappedNums = (array) => {
+//     return array.map(value => {
+//         return value * 5
+//     })
+// }
 
+// console.log(mappedNums(myMapNumbers2))
 
+// I THINK I GOT IT!!!! I WAS HAVING TROUBLE FIGURING OUT WHERE TO PUT MY (ARRAY) AND WHERE TO PUT MY (VALUE)
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
-
+const removedVowels = (array) => {
+// // 	// Create variable to store the letters
+    let letters = array.split(""); 
+    let letters2 =letters.filter(value => {
+        if (! ["a", "e", "i", "o", "u"].includes(value)) {
+            return true
+        } else {
+            return false
+        }
+    })
+    return letters2.join('')
+}
+console.log(removedVowels(removedVowels(stringWithVowels2)))
+// //         // create an argument that check to see if statements are true
+//             // if true, then it will push whatever does not match the vowels provided
+              
+// // Expected output "HyThrLrnStdnt" "LvJvScrpt"
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string" "mCdngMstr"
 
-var refactorTester1 = true
-var refactorTester2 = 42
-var refactorTester3 = "IAmACodingMaster"
-
-
+// var refactorTester1 = true
+// var refactorTester2 = 42
+// var refactorTester3 = "IAmACodingMaster"
+// // pasted the previous syntax from questions 4.
+// // need function to include nonstreing edges
+// const removedVowels = (array) => {
+//         let letters = array.split(""); 
+//         let letters2 =letters.filter(value => {
+//             // need it to check for uppercase and lowercase letters
+//             if (! ["a", "e", "i", "o", "u"].includes(value)) {
+//                 // need it to check nonstring edges
+//                 return true
+//             } else {
+//                 return false
+//             }
+//         })
+        
+//  if (typeof value === "number" || typeof value === "boolean" {
```

Comment:
nice! I like the use of `||` operator to reduce the number of else-ifs!
```suggestion
//  if (typeof value === "number" || typeof value === "boolean") {
```